src/pyasm/security/security.py

```diff
@@ -790,7 +790,7 @@ def get_login_wdg(cls, hash=None):
 
 
     def allow_guest(cls, url=None):
-        return False
+        return None
```

Comment:
Can you explain the difference between None and False.   The name of the function suggests a True / False return.